recipes/varscan/2.3.7/varscan.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
Hmm..I suppose you meant the Java wrapper? In that case, I see that the existing `varscan.sh` script seems to conform the guideline still. Am I perhaps missing something?